src/components/Dashboard.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+import DashNav from './DashNav'
+import LocalNav from './LocalNav'
+import DashBody from './DashBody'
+
+class Dashboard extends Component {
+  render() {
+    return (
+      <div className="dashboard-home">
+        <DashNav/>
```

Comment:
Things to rename: 

- DashNav -> DashboardNav
- DashBody -> DashboardBody
- DashWidget -> DashboardWelcome